aten/src/ATen/native/quantized/cpu/qnnpack/test/dwconv-microkernel-tester.h

```diff
@@ -205,14 +207,24 @@ class DWConvMicrokernelTester {
 
       std::fill(packedWeights.begin(), packedWeights.end(), 0xA5);
 
+      size_t num_zero_points_padded = channels() + 8;
```

Comment:
Is the +8 for alignment?

Reply:
Since access can be out of bounds otherwise in corner cases.